src/pages/pr/collage_template.tsx

```diff
@@ -0,0 +1,110 @@
+import { Box, Divider, HStack, Image, Text, VStack } from "@chakra-ui/react";
+
+interface CollageProps {
```

Comment:
[imo] 特に強いこだわりはありませんが、
このプロジェクトでPropsを指定する場合は`interface`ではなく、`type`を使っています。
```suggestion
type Props {
```

`interface`は`class`のメソッドを指定するときに使うイメージがあります。